builtin/mainmenu/content/contentdb.lua

```diff
@@ -93,11 +93,27 @@ local function start_install(package, reason)
 		if result.msg then
 			gamedata.errormessage = result.msg
 		else
+			local delete_old_dir
+			if package.path then
+				local path_array = package.path:split(DIR_DELIM)
```

Comment:
Since you need just the last part, why not do `local basename = package.path:match("[^/\\]+$")`?